Dockerfile

```diff
@@ -41,7 +41,7 @@ COPY --from=builder /tmp/ipxe/src/bin/undionly.kpxe /tftpboot
 COPY --from=builder /tmp/ipxe/src/bin-x86_64-efi/snponly.efi /tftpboot
 COPY --from=builder /tmp/ipxe/src/bin-x86_64-efi/ipxe.efi /tftpboot
 
-COPY --from=builder /tmp/esp.img /httpboot/uefi_esp.img
+COPY --from=builder /tmp/esp.img /tmp/uefi_esp.img
```

Comment:
could we copy this directly to /shared/html/uefi_esp.img ?

Reply:
No, /shared is a shard volume that is shared between all of the ironic containers, it only gets created when the containers are started.